pkgs/top-level/all-packages.nix

```diff
@@ -15893,11 +15893,6 @@ with pkgs;
     sdk = true;
   };
 
-  # only kept for nixui, see https://github.com/matejc/nixui/issues/27
-  nwjs_0_12 = callPackage ../development/tools/node-webkit/nw12.nix {
```

Comment:
I think you forgot to delete the folder `pkgs/development/tools/node-webkit`

Reply:
Indeed, this was not removed. Fixed it.